ledger/src/blockstore_metric_report_service.rs

```diff
@@ -26,15 +26,24 @@ impl BlockstoreMetricReportService {
     pub fn new(blockstore: Arc<Blockstore>, exit: Arc<AtomicBool>) -> Self {
         let t_cf_metric = Builder::new()
             .name("solRocksCfMtrcs".to_string())
-            .spawn(move || loop {
-                if exit.load(Ordering::Relaxed) {
-                    break;
+            .spawn(move || {
+                info!("BlockstoreMetricReportService has started");
+                let mut last_report_time = Instant::now();
+                loop {
+                    if exit.load(Ordering::Relaxed) {
+                        break;
+                    }
+
+                    if last_report_time.elapsed() > BLOCKSTORE_METRICS_REPORT_INTERVAL {
+                        blockstore.submit_rocksdb_cf_metrics_for_all_cfs();
+                        blockstore.report_rpc_api_metrics();
+
+                        last_report_time = Instant::now();
+                    }
+
+                    thread::sleep(Duration::from_secs(1));
```

Comment:
Might be nice to set the sleep interval dynamically before entering the loop in case we change (reduce) the metrics reporting interval later. E.g. make it the min of 1 second and the metrics reporting interval

Reply:
Fair, I'm going to save that for the hypothetical subsequent PR where we reduce reporting interval